scripts/cloud/aws/ops-ec2-snapshot-ebs-volumes.py

```diff
@@ -76,23 +78,24 @@ def main(self):
 
 
```

Comment:
Consider adding an "is_region_valid" method to check that the --region param is valid.